File: app/src/main/java/com/mobilegiants/megila/LawsOfPurimActivity.java
package com.mobilegiants.megila;

import android.app.Activity;
import android.os.Bundle;

public class LawsOfPurimActivity extends Activity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		try {
			setContentView(R.layout.activity_laws_of_purim);
		} catch (Exception e) {

			e.printStackTrace();
			setContentView(R.layout.activity_laws_of_purim_2);
		}
	}

	

}

File: app/src/main/java/com/mobilegiants/megila/LawsOfPurimActivity.kt
package com.mobilegiants.megila

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class LawsOfPurimActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        try {
            setContentView(R.layout.activity_laws_of_purim)
        } catch (e: Exception) {
            e.printStackTrace()
            setContentView(R.layout.activity_laws_of_purim_2)
        }
    }
}
